src/ResourceManager/Sql/Commands.Sql/Database Backup/Cmdlet/RemoveAzureSqlDatabaseRestorePoint.cs

```diff
@@ -0,0 +1,78 @@
+// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Sql.Backup.Model;
+using System.Collections.Generic;
+using System.Management.Automation;
+using System;
+
+namespace Microsoft.Azure.Commands.Sql.Backup.Cmdlet
+{
+    [Cmdlet(VerbsCommon.Remove, "AzureRmSqlDatabaseRestorePoint", SupportsShouldProcess = true,
+        ConfirmImpact = ConfirmImpact.None)]
```

Comment:
Same comments here about outputtype and confirmimpact.

Reply:
added